config.dev.yaml

```diff
@@ -102,3 +102,7 @@ services:
       # If true, runs this instance without processing tiles
       # This could be good for queue management
       uiOnly: false
+
+      # Set different timeout for tile generation in case mapnik stuck on
+      # locked resource while reading Postgres
+      tileTimeOut: 90000
```

Comment:
90000 is fine for a dev/example config, but for the WMF production config (defined by template in tilerator/deploy), I think we'll need to set a value lower than the heartbeat monitoring timeout ([7500](https://github.com/wikimedia/puppet/blob/ae134dda6e0f754db347e659adf7c75cede672d6/modules/service/manifests/node/config/scap3.pp#L14), if I'm reading the right value).  Otherwise the heartbeat monitor will kill the process before the promise has a chance to time out.